### DISCUSSION TITLE
Approved new formula not being merged

### DISCUSSION THREAD

**Discussion by boris-kolpackov** [Homebrew/discussions#4812] [Writing Formulae/Casks] [answered]
### Output of `brew config`

```shell
n/a
```

### Output of `brew doctor`

```shell
n/a
```

### Description of issue

I have the (new) [`build2` formula PR](https://github.com/Homebrew/homebrew-core/pull/141550) sitting in the approved stage since Sep 20 but not being merged for some reason (even though I can see later submissions of new formulas are being merged). I read in [How To Open a Homebrew Pull Request](https://docs.brew.sh/How-To-Open-a-Homebrew-Pull-Request) that "Post a comment on your pull request if you’ve provided all the requested changes/information and it hasn’t been merged after a week", which I did but got no response. I feel like I am likely missing something so basic here that it doesn't warrant a response from the maintainers. Could perhaps someone from the community help me out here? Perhaps some bottles need to be pre-built before a new formula can be merged? 

Sorry for being impatient but that PR has been dragging for over three weeks now and this open loop is driving me a bit crazy.

Thanks in advance for any feedback!

**Comment by p-linnane** [comment, marked as answer]
We have to manually merge new formulae once approved. This one was missed, as we are in the middle of trying to rebuild every formula for Sonoma and spread thin. I will merge it now, thanks for your patience.

**Reply by boris-kolpackov** [reply]
Thanks for the clarification (it makes perfect sense) and looking into the PR!
